content/en/metrics/volume.md

```diff
@@ -0,0 +1,120 @@
+
+## Compare a metric's cardinality (volume) over time 
+
+{{< img src="metrics/volume/compare_metric_cardinality.png" alt="Metrics Volume filtered down to metric names with “shopist”, sorted by estimated custom metrics. On hover over the change in volume, displays the cardinality graph of the metric over the past day" style="width:100%;" >}}
+
+When identifying your top 500 metric names by change in volume, you can additionally hover over the number to compare a metric name's # of indexed custom metrics (its cardinality) over time. As a reminder, a single metric name can emit multiple indexed custom metrics (quick refresher on how we meter and bill for custom metrics here[6]) 
+
+To compare your spiking metric's cardinality over time:
+1. Select a time frame in the top right hand corner (the recommended time frame is **Past 1 Day** or **Past 4 Weeks**).
+2. Select the metric name which you want the view the cardinality over time and in the same row click on the value under the **Change in Volume** column. This opens up a modal showing a graph comparing your metric's cardinality over time and the percentage increase in its spike.
+3. (Optional) Create a Change monitor for `% change` to proactively alert on this spiking metric. For more information, see the [Change Alert Monitor][2] documentation.
+
+## Identify less valuable, unqueried metrics
+
+{{< img src="metrics/volume/id_unqueried_metrics.png" alt="Facet fields for Query Activity with the 'Not actively queried' facet selected" style="width:100%;" >}}
+
+To start reducing custom metrics costs, organizations often start with their largest metric names that aren't valuable to the organization; in other words, ones that aren't actively queried. Datadog's intelligent query insights analyze your queries and surfaces your unqueried metrics over the past 30 days. Our analysis is constantly running in the background ensuring that your unqueried metrics are always up-to-date and available self-service.
+
+To find the metrics not actively queried in the past 30 days, click on **Not Actively Queried** in the *Query Activity Facet* box. Selecting **Not Actively Queried** generates a list of unused metric names across dashboards, notebooks, monitors, SLOs, Metrics Explorer, and the API.
+
+## How to quickly reduce metric volume and cost
+
+After you identify unqueried metrics, you can quickly and confidently eliminate the volume and cost of these metric names by using [Metrics without Limits™][3] without a single line of code. By using Metrics without Limits, you ensure that you pay only for the metrics that you use by eliminating timeseries that are never or rarely leveraged. Based on our intelligent query insights, the average customer's custom metrics volume can be reduced by 70% if they were to use Metrics without Limits on their unqueried metric names. 
```

Comment:
would prefer to keep this stat where it is because it only applies to unqueried metrics and isn't really part of the value prop of the volume page itself. The volume page focuses on the subset of largest metrics that are unqueried; while if a user were to use MWL on all unqueried metrics (not just that of the top 500) they'd could see up to 70% reduction.

Reply:
Can this stat be moved to the guide or blog? I understand why you want readers to know about this, but I'm not sure the main docs are the right place for this. Sounds like we're trying to convince a reader they need a product rather than telling them what this product can do

```suggestion
After you identify unqueried metrics, you can eliminate the volume and cost of these metric names by using [Metrics without Limits™][3] without a single line of code. By using Metrics without Limits, you ensure that you pay only for the metrics that you use by eliminating timeseries that are never or rarely leveraged. Use Metrics without Limits™ on your unqueried metric names to reduce custom metrics volume. 
```